website/client/src/components/settings/resetModal.vue

```diff
@@ -37,11 +37,8 @@ export default {
       this.$root.$emit('bv::hide::modal', 'reset');
     },
     async reset () {
-      const response = await axios.post('/api/v4/user/reset');
-      // @TODO: Not sure if this is correct
-      this.$store.user = response.data.data.user;
-      this.$router.push('/');
-      this.$root.$emit('bv::hide::modal', 'reset');
+      await axios.post('/api/v4/user/reset');
```

Comment:
@PitiTheGrey the new changes are working great but I just realized that this way you will not get redirected to the tasks page like the original code,

```
      this.$router.push('/');
      setTimeout(() => window.location.reload(true));
```

^ should fix it, this way you first go to the tasks page and then it immediately reloads the page (the setTimeout is needed to make sure the reload doesn't happen before you reach the tasks page)